pkgs/by-name/re/recuperabit/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "recuperabit";
+  version = "1.1.6-unstable-2025-03-02";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "Lazza";
+    repo = "RecuperaBit";
+    rev = "f762099345ba8a6c7388253ee8c0c6b2284d30b9";
+    hash = "sha256-1johhRBvFA3PszoEAulwnhV1ApsedzIC0aJSgUrgUX4=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share
```

```suggestion
    mkdir -p $out/bin
    mkdir -p $out/share
```

I can build after creating directory again